src/views/concepts/ConceptPage.jsx

```diff
@@ -2,21 +2,22 @@ import React from 'react';
 
 import BlueLinkButton from 'shared/buttons/BlueLinkButton';
 import Accordion from '../../shared/accordion/Accordion';
-import styles from './ConceptPage.module.scss';
 import { appRoutes } from 'constants.js';
 import Footer from '../../components/footer/Footer';
+import ConceptPageStyle, { ContentStyle } from './ConceptPageStyle';
+import { IntroTextStyle } from '../../styles/SharedStyles';
 
 const ConceptPage = () => {
   return (
-    <div className={styles.Container}>
-      <div className={styles.Content}>
+    <ConceptPageStyle className="ConceptPageStyle">
+      <ContentStyle className={'ContentStyle'}>
```

Comment:
For consistency's sake, I would use `"ContentStyle"` instead of `{'ContentStyle'}`

Reply:
thanks, the classNames can be left out here